internal_worker.go

```diff
@@ -40,9 +40,15 @@ import (
 )
 
 const (
-	defaultConcurrentPollRoutineSize          = 2
-	defaultMaxConcurrentActivityExecutionSize = 10000  // Large execution size(unlimited)
-	defaultMaxActivityExecutionRate           = 100000 // Large execution rate(100K per sec)
+	// Set to 1 for now, can adjust later if needed. The typical RTT (round-trip time) is below 1ms within data center.
+	// With 1 poller, we could easily achieve a few hundred RPS.
+	defaultConcurrentPollRoutineSize = 1
+
+	defaultMaxConcurrentActivityExecutionSize = 1000   // Large concurrent activity execution size (1k)
+	defaultMaxActivityExecutionRate           = 100000 // Large activity execution rate (unlimited)
+
+	defaultMaxConcurrentWorkflowExecutionSize = 10     // workflow code normally run fast, so only need small concurrent number
```

Comment:
true. May be set this to 100? This is a value currently not exposed to user, and is hard coded here.